### PR TITLE
New README file!

### PR DESCRIPTION
**Pull request by darkdarcool**
Added a readme.md file so people can understand the module better!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# REPLAPI Module
+
+Hi! And welcome to the REPLAPI(pronouned "rep-"ll"-"a"-"p"-"i") module!
+
+This module acceses your(and anyone elses replit stats)!
+
+This gives any users `BIO`, `NICKNAME`, and `FAVORITE LANGS`.
```
Comment (JBYT27) [inline README.md:7]: Uh, im planning on adding more, but ok

**Comment by JBYT27** [comment]
Hmmm... Well i already have a readme file, but ill add that on. :D

File: README.md (hunk 1 above)
Comment (darkdarcool) [reply README.md:7]: ok, this is for GitHub, like a readme they opens when you open the repo

Comment (darkdarcool) [reply README.md:7]: and yeh, you can edit the readme later

**Pull request merged**